src/lib/mathlib/math/filter/LowPassFilter2pArray.hpp

```diff
@@ -54,7 +54,7 @@ class LowPassFilter2pArray : public LowPassFilter2p
 	 *
 	 * @return retrieve the filtered result
 	 */
-	inline float apply(const int16_t samples[], uint8_t num_samples)
+	inline float apply(const float samples[], uint8_t num_samples)
```

Comment:
I know this is ugly, but it was done to avoid copying all the data again. Let me see if it matters (we're barely using the FIFOs currently).

Reply:
I had to do that because I wanted the NotchFilterArray template to output a float array, so then the lowpass needs to accept a float array as well